res/values/styles.xml

```diff
@@ -14,6 +14,7 @@
         <item name="android:windowFullscreen">true</item>
         <item name="android:windowBackground">#000</item>
         <item name="android:forceDarkAllowed" tools:targetApi="q">false</item>
+        <item name="android:windowLayoutInDisplayCutoutMode" tools:targetApi="27">shortEdges</item>
```

Comment:
I'm not overly sure with this change. From the result of my testing in my Android 15 device, it seemed to have worsen the situation instead (the padding is now bigger regardless of the cutout area). In addition to that, it rendered the game *in* the cutout area itself, which is not good given that the cutout area typically has a lower touch sensitivity.

I considered changing this to `never` as per [Android's documentation](https://developer.android.com/develop/ui/views/layout/display-cutout#never), but it is also noted that this is not supported in Android 15 for non-floating windows, so... yikes.

Reply:
Maybe we can try using real display metrics rather than what we're using currently.

Reply:
Can you try how it looks with latest changes ?

Reply:
There's no more padding. However, the game still renders in the cutout area. I think the real width and height should account for the cutout area.

Reply:
It should but I think that should be managed by the game UI.
We can compute the cutout area and offset elements/scenes according to it.

Reply:
Exactly, but I consider that what we have is enough (HUD editor).